test/functional/page_objects/settings_page.ts

```diff
@@ -384,9 +384,14 @@ export class SettingsPageObject extends FtrService {
   }
 
   async filterField(name: string) {
-    const input = await this.testSubjects.find('indexPatternFieldFilter');
+    const input: WebElementWrapper = await this.testSubjects.find('indexPatternFieldFilter');
```

Comment:
I don't think we need to explicitly define type here, did I something?

Reply:
Yes, but I like it for later.  But, will drop! 😉

Reply:
https://github.com/elastic/kibana/pull/194738/commits/b0ec24fd269738a0de0a3d9363bd3224e4028cb6